src/main/java/nextstep/subway/line/ui/LineController.java

```diff
@@ -37,31 +36,26 @@ public ResponseEntity createLine(@RequestBody LineRequest lineRequest) {
     }
 
     @PutMapping("/{id}")
-    public ResponseEntity updateLine(@PathVariable Long id, @RequestBody LineRequest lineUpdateRequest) {
+    public ResponseEntity<LineResponse> updateLine(@PathVariable Long id, @RequestBody LineRequest lineUpdateRequest) {
```

Comment:
```suggestion
    public ResponseEntity<Void> updateLine(@PathVariable Long id, @RequestBody LineRequest lineUpdateRequest) {
```

응답의 body가 없다면 타입으로 의도를 나타낼 수 있습니다!

Reply:
네 타입으로 의도를 나타낼수 있어서 직관적입니다!! 수정하겠습니다!